.gitignore

```diff
@@ -38,4 +38,9 @@ terraform
 # Ignore the Go vendor directory, superseded by proxy.golang.org and go.{mod,sum}
 vendor
 
+# ignore binaries in the root dir
 terraform-provider-iterative
+leo-*
+
+# JetBrains IDEs
+.idea/*
```

Comment:
Really dislike adding personal things to `.gitignore`. People's personal setups should use `.git/config/exclude` etc. instead

Reply:
I disagree here I'm afraid 🙏 😄  - it's an assumption that is very problematic. I also suggest to add common IDE `.gitignore`s to any other repo if missing actually, there aren't many common IDE stuff to ignore to cover 95% of cases (super niche editors and custom metadata set-ups aren't relevant).

I think this an opinion vs pragmatism scenario;

Even if it would've been nice to have people systematically use personal configs in theory (good mechanism, that's what its for, might be nice to separate setup-based-ignores and project-based-ignores), in practice, most developers just don't bother with this.
It's enough that those exist in non-negligable numbers to justify removing this as a barrier imo.

Especially a must for open source repos IMO - We can't assume those things on external contributors even if we wanted to assume+enforce using personal git profiles inside a company.

For those who do use personal gitignores - this is harmless, for those who don't - omitting this for cleanliness/aesthetics reason will add needless friction which is real.



To illustrate/support:
- internal example - [dvc's gitignore](https://github.com/iterative/dvc/blob/main/.gitignore)
- external example for something that I know most people in iterative like - [fastapi's .gitignore](https://github.com/tiangolo/fastapi/blob/master/.gitignore)

Reply:
Another reason just occurred to me (re: cloud IDE) - the case for ephemeral dev boxes

Reply:
I think ide-specific exclude paths are perfectly fine in .gitignore. It's a low impact change that we're spending an unreasonable amount of time discussing.